numba/compiler.py

```diff
@@ -666,6 +677,41 @@ def _compile_bytecode(self):
             assert self.cr is not None
             return self.cr
 
+    def _compile_ir(self):
```

Comment:
addressed in https://github.com/numba/numba/pull/1996/commits/4bebf7a743a3c53eef16956a822977d61f8cd4b4